samples/stack-overflow/index.ts

```diff
@@ -12,6 +12,8 @@ const app: ReadableApp<string> = async function (_stream: any, interval: number,
         if (Object.keys(result.diff).length > 0) outputStream.write(result);
     }, interval * ONE_MINUTE);
 
+    (outputStream as SynchronousStreamable<any>).contentType = "application/x-ndjson";
```

Comment:
What do you think if we added the comment above this line explaining what the content type setting is for?
e.g.
`# If you wish you operate the output stream in "object mode", please specify the content type of the returned stream to be "application/x-ndjson". You can implement by using SynchronousStreamable type from "@scramjet/types".`

Reply:
Thank you Ala! I totally forgot about it. Comment added as suggested :)